src/java.base/share/classes/java/util/ArrayDeque.java

```diff
@@ -351,6 +351,50 @@ public boolean offerLast(E e) {
         return true;
     }
 
+    // *** Indexed methods ***
+
+    /**
+     * Returns the element at the specified position in this deque.
+     *
+     * @param index index of the element to return
+     * @return the element at the specified position in this deque
```

Comment:
This can use the inline form of `@return`:
```suggestion
     * {@return the element at the specified position in this deque}
     *
     * @param index index of the element to return
```